gio/src/auto/file.rs

```diff
@@ -268,30 +268,12 @@ pub trait FileExt: IsA<File> + sealed::Sealed + 'static {
         }
     }
 
-    #[cfg(feature = "v2_82")]
-    #[cfg_attr(docsrs, doc(cfg(feature = "v2_82")))]
-    #[doc(alias = "g_file_copy_async_with_closures")]
-    fn copy_async_with_closures(
-        &self,
-        destination: &impl IsA<File>,
-        flags: FileCopyFlags,
-        io_priority: i32,
-        cancellable: Option<&impl IsA<Cancellable>>,
-        progress_callback_closure: Option<&glib::Closure>,
-        ready_callback_closure: &glib::Closure,
-    ) {
-        unsafe {
-            ffi::g_file_copy_async_with_closures(
-                self.as_ref().to_glib_none().0,
-                destination.as_ref().to_glib_none().0,
-                flags.into_glib(),
-                io_priority,
-                cancellable.map(|p| p.as_ref()).to_glib_none().0,
-                progress_callback_closure.to_glib_none().0,
-                ready_callback_closure.to_glib_none().0,
-            );
-        }
-    }
+    //#[cfg(feature = "v2_82")]
+    //#[cfg_attr(docsrs, doc(cfg(feature = "v2_82")))]
+    //#[doc(alias = "g_file_copy_async_with_closures")]
```

Comment:
The fix for this still hasn't landed on the gir file? cc @A6GibKm

Reply:
https://gitlab.gnome.org/GNOME/glib/-/merge_requests/4099 I think it is only in main, I don't think I have updated this gir file on gir-files.

Reply:
Latest update to the gir at gir-files was 2 weeks ago, patch was merged a week ago, so no, it is not in there.

Reply:
I see it here `   <method name="copy_async_with_closures" c:identifier="g_file_copy_async_with_closures" shadows="copy_async" version="2.82" glib:finish-func="copy_finish">`

Reply:
From a quick look, it seems like the finish-func annotation is wrongly set in that function. It probably needs a manual implementation to turn it into a future instead of still using closures.

Reply:
@bilelmoussaoui can you ping ebassi about that? Would be good to get this in to not have this PR bitrot